ulohy_07/1D_piskvorky/test_piskvorky.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import piskvorky
+import util
+import ai
+
+# Testování funkce, která vyhodnotí výsledek
+def test_vyhodnoceni1():
+    assert piskvorky.vyhodnot("----xxx-------------") == "x"
+
+def test_vyhodnoceni2():
+    assert piskvorky.vyhodnot("----x-x------ooo----") == "o"
+
+def test_vyhodnoceni3():
+    assert piskvorky.vyhodnot("xxooxxooxxooxxooxxoo") == "!"
```

Comment:
Testy dobré, jen to pojmenování funkcí by mohlo dávat víc smysl. Když to bude velký program a testů bude 20 tak bude mít každý stejný název a svoje číslo? Lepší je vždycky pojemenovávat proměnné, funkce, testy i objekty tak, aby dávaly smysl v kontextu programu a své role a už samy napovídali, co se tam děje i bez jáké hokoliv komentáře. Zde to je sice jasné je, protože jsou 3 na 6 řádků ale je dobré si dělat dobré návyky už od základů. Např. testy `test_x_won`, `test_check_winner_x` nebo něco podobného.

Reply:
Tak na tomto ještě musím asi hodně zapracovat, abych pojmenovávala srozumitelněji. Opraveno.